File: 771.cpp
/*
	Author: Zoljargal Gantumur
	Runtime: 0 ms, faster than 100.00% of C++ online submissions
	Memory Usage: 6.1 MB, less than 92.04% of C++ online submissions
*/

class Solution {
public:
    int numJewelsInStones(string J, string S) {
        int count = 0;
        int lenJ, lenS;
        lenJ = J.size();
        lenS = S.size();
        for(int i=0; i<lenS; i++){
            for(int j=0; j<lenJ; j++){
                if(S[i] == J[j]){
                    count++;
                }
            }
        }
        return count;
    }
};

File: 771.py
# Author: Zoljargal Gantumur
# Runtime: 36 ms, faster than 30.58% of Python3 online submissions
# Memory Usage: 13.7 MB, less than 86.53% of Python3 online submissions

class Solution:
    def numJewelsInStones(self, J: str, S: str) -> int:
        count = 0
        for i in range(len(S)):
            for j in range(len(J)):
                if S[i] == J[j]:
                    count += 1
        return count

